static/include/how-to/gather-sync-sensor.md

```diff
@@ -0,0 +1,48 @@
+Viam's [data management service](/services/data/) lets you capture data locally from sensors and then sync it to the cloud where you can access all data across different {{< glossary_tooltip term_id="machine" text="machines" >}} or {{< glossary_tooltip term_id="location" text="locations" >}} and see trends over time.
+
+{{< table >}}
+{{< tablestep link="/services/data/">}}
+{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure the data management service">}}
+**1. Add the data management service**
+
+On your machine's **CONFIGURE** tab, add the **data management** service.
+
+Enable **Syncing** to ensure captured data is synced to the cloud and set the sync interval, for example to `0.05` minutes to sync every 3 seconds.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/services/icons/data-management.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Configure the data management service">}}
+**2. Tag your data**
+
+Add a tag to your machine's captured data, for example `machine1-sensor-data`. This makes querying data easier because you can query by tag.
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/icons/components/sensor.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="configure a camera component">}}
+**3. Capture data from sensor**
+
+On the **CONFIGURE** tab, go to the **sensor**'s card and find the **Data capture** section.
+Add a new method, `Readings`, to capture data for and set the frequency, for example to `0.1` readings per second. This will capture data once every ten seconds.
```

Comment:
Looks like there's an extra "for"
```suggestion
Add a new method, `Readings`, to capture sensor readings and set the frequency, for example to `0.1` readings per second.
This will capture data once every ten seconds.
```
Seems like this could be less hand-holdy or at least more succinct since the app says what the units are.
Briefest:
```suggestion
Add a new method, `Readings`, and set the frequency at which to capture sensor readings.
```
Medium-brief:
```suggestion
Add a new method, `Readings`, to capture data for and set the frequency.
For example, setting a frequency of `0.1` will capture data once every ten seconds.
```